src/noise_interfaces/noise_process_interface.jl

```diff
@@ -443,7 +447,11 @@ end
       end
     end
   else # Bridge
-    i = searchsortedfirst(W.t,t)
+    if reverse
```

Comment:
if W.reverse and don't need the keyword argument?

Reply:
but W is the `NoiseProcess` and not the `NoiseWrapper`.. so it doesn't have the keyword reverse.

Reply:
oh makes sense